chromium/browser_type.go

```diff
@@ -387,11 +387,11 @@ func execute(
 		}()
 
 		if err := cmd.Wait(); err != nil {
-			logger.Errorf("BrowserType:execute", "cmd.Wait: %v", err)
-		}
-
-		if err := stdout.Close(); err != nil {
-			logger.Errorf("BrowserType:execute", "stdout.Close: %v", err)
+			if err.Error() != "signal: killed" {
+				logger.Errorf("BrowserType:execute", "browser process with PID %d ended unexpectedly %v", cmd.Process.Pid, err)
+			} else {
+				logger.Debug("BrowserType:execute", "browser process with PID %d killed, context cancelled", cmd.Process.Pid)
```

Comment:
I think we can use the same log message in both cases, just change the method to call. And we don't need to mention `context cancelled`; we can't know whether that's always the case, so it might be confusing.

So something like:
```go
if err := cmd.Wait(); err != nil {
	log := logger.Errorf
	if err.Error() == "signal: killed" {
		// The browser process is killed when the context is cancelled
		// after a k6 iteration ends, so silence the log message until
		// we can stop it gracefully. See #new-issue-number
		log = logger.Debugf
	}
	log("BrowserType:execute", "browser process with PID %d ended unexpectedly: %v", cmd.Process.Pid, err)
}
```

Reply:
Resolved in: 85046600c42120189084442e6e56a3caa5096bd1